x-pack/plugins/aiops/server/routes/explain_log_rate_spikes.ts

```diff
@@ -132,7 +136,8 @@ export const defineExplainLogRateSpikesRoute = (
         try {
           fieldCandidates = await fetchFieldCandidates(client, request.body);
         } catch (e) {
-          push(addErrorAction(e.toString()));
+          logger.error(`Failed to fetch field candidates, got: \n${e.toString()}`);
+          pushError(`Failed to fetch field candidates.`);
```

Comment:
nit: Should these error messages be internationalized?

Reply:
We have an item in the meta issue for 8.6 to pick this up, it affect all messages in this file, not just the updated ones, will do in a follow up: https://github.com/elastic/kibana/issues/141214